src/util/log.d

```diff
@@ -86,7 +87,7 @@ bool disabled(LogLevel level) pure
     return (level & levels) != 0;
 }
 
-private alias Sink = OutputRange!(const(char)[]);
+private alias Sink = Algebraic!(BufferedTextWriter!(File.LockingTextWriter)*, Appender!string);
```

Comment:
That's awful!

The `format` documentation states that the `delegate` is "accepted for legacy reasons or for use in virtual functions". We should either get rid of the virtual functions or stick to the `delegate`.